templates/TCA with ReducerProtocol.xctemplate/___FILEBASENAME___View.swift

```diff
@@ -30,14 +30,11 @@ public struct ___FILEBASENAMEASIDENTIFIER___: View {
 
 // MARK: - Preview
 
-struct ___FILEBASENAMEASIDENTIFIER____Previews: PreviewProvider {
-  static var previews: some View {
-    ___FILEBASENAMEASIDENTIFIER___(store: store)
-      .previewLayout(.sizeThatFits)
-  }
+let store: StoreOf<___VARIABLE_productName:identifier___> = .init(
+  initialState: .init(),
+  reducer: { ___VARIABLE_productName:identifier___() }
+)
```

Comment:
Since the global variable store is only for preview,
I think It's better to wrap in `#if DEBUG` 
or inline in `___FILEBASENAMEASIDENTIFIER___(store: ....)`

Reply:
Since some might not use #if DEBUG flags, moved init to inline: 
[ef4285](https://github.com/riiid/xctemplate-for-tca/pull/5/commits/4ef42852c0838a7d40f867b95f951347d98cf7ff)